e2e/workflow/installation.go

```diff
@@ -106,6 +106,36 @@ func (w *InstallationSuite) CreateInstallation(ctx context.Context) error {
 	return nil
 }
 
+// CreateInstallation creates new Installation and waits for it to reach stable state.
+func (w *InstallationSuite) CreateInstallationWithVersionedAWSS3Filestore(ctx context.Context) error {
+	installationBuilder := pkg.NewInstallationBuilderWithDefaults().
+		DNS(pkg.GetDNS(w.dnsSubdomain)).
+		DB(w.Params.DBType).
+		FileStore(model.InstallationFilestoreAwsS3).
+		Annotations(w.Params.Annotations)
+
+	installation, err := w.client.CreateInstallation(installationBuilder.CreateRequest())
+	if err != nil {
+		return errors.Wrap(err, "while creating installation")
+	}
+	w.logger.Infof("Installation created: %s", installation.ID)
+	w.Meta.InstallationID = installation.ID
```

Comment:
One side effect of this is the test webhook that is sent after the run will probably only contain the log link for this installation and not the first installation in the suite. This isn't a big deal. I want to refactor the webhooks anyway so just noting it here if we end up using the log links after this is merged.